packages/suite/src/hooks/wallet/__fixtures__/useRbfForm.ts

```diff
@@ -71,7 +74,26 @@ const BTC_CJ_ACCOUNT = {
 //     // script_type: 'PAYTOADDRESS',
 // },
 
-const PREPARE_TX = (params = {}) => ({
+const txDummyData = {
+    deviceState: 'A@B:1',
+    descriptor: '',
+    type: 'sent',
+    txid: '',
+    amount: '',
+    fee: '',
+    targets: [],
+    tokens: [],
+    internalTransfers: [],
+    details: {
+        vin: [],
+        vout: [],
+        size: 0,
+        totalInput: '',
+        totalOutput: '',
+    },
+} satisfies Partial<WalletAccountTransaction>;
```

Comment:
just btw, are you familiar with [@total-typescript/shoehorn](https://www.npmjs.com/package/@total-typescript/shoehorn)? I have used it before and I do like the approach. This'd be a good opportunity to try it out..

Reply:
Discussed, we don't see advantage in it.